exporters/otlp/metrics/src/main/java/io/opentelemetry/exporter/otlp/metrics/OtlpGrpcMetricExporterBuilder.java

```diff
@@ -51,21 +50,6 @@
             GRPC_ENDPOINT_PATH);
   }
 
-  /**
-   * Sets the managed chanel to use when communicating with the backend. Takes precedence over
-   * {@link #setEndpoint(String)} if both are called.
-   *
-   * @param channel the channel to use
-   * @return this builder's instance
-   * @deprecated Use {@link #setEndpoint(String)}. If you have a use case not satisfied by the
-   *     methods on this builder, please file an issue to let us know what it is.
-   */
-  @Deprecated
-  public OtlpGrpcMetricExporterBuilder setChannel(ManagedChannel channel) {
-    delegate.setChannel(channel);
-    return this;
-  }
```

Comment:
I would understand the argument to keep this to maintain symmetry with `OtlpGrpcSpanExporterBuilder`.

Reply:
Yeah let's go ahead and keep this one